scripts/ta_bin_to_c.py

```diff
@@ -6,8 +6,10 @@
 
 import argparse
 import array
+from elftools.elf.elffile import ELFFile
```

Comment:
I think to remember `from XXX import YYY` was not recommended.
However I'm far from a python master so i leave it up to @jforissier or @jenswi-linaro to status on this.

Reply:
Honestly I have no idea :wink: I can just say I find the `from XXX import YYY` syntax quite convenient, but if there are good reasons not to use it I'll be happy to hear them!

Reply:
Fine then, discard my comment :confused:  _<--- this is expected to be a confused face_

Reply:
here is does not matter but I think import sub library is faster then library. Also it's nice to see that for example after library upgrade method was changed and import generates exception on early stage.